### PR TITLE
Highlight active category, use borders on mobile too (bug 1166413)

### PR DESCRIPTION
**Pull request by mstriemer**
https://bugzilla.mozilla.org/show_bug.cgi?id=1166413

category-active-mobile mov

### CODE REVIEW

**Comment by sherlockedci** [comment]
[ The Adventure of the Beryl Build](http://sherlocked.dev.mozaws.net/builds/65582298)

File: src/media/css/elements--categories.styl [hunk 1]
```diff
@@ -34,6 +36,9 @@ mkt-category-item {
     padding: 10px;
     transition: none;
 
+    +media--wide-tablet() {
```
Comment (spasovski) [inline src/media/css/elements--categories.styl]: why not in `@media $wide-tablet`? if it's not defined we can

**Comment by ngokevin** [comment]
borders on mobile are a bit weird, but if that's we call for r+

File: src/media/css/elements--categories.styl (hunk 1 above)
Comment (mstriemer) [inline src/media/css/elements--categories.styl]: I prefer it closer to the base styles like this. I thought we had decided to use this for bidi/responsive stuff now.

**Pull request merged**